src/flows/data_retention/deleted_accounts_flow.py

```diff
@@ -20,8 +20,8 @@ def query_file(file_name: str, **kwargs):
     )
 
 
-# with Flow(FLOW_NAME, schedule=get_cron_schedule(cron="0 0 1 * *")) as flow:
-with Flow(FLOW_NAME) as flow:
+with Flow(FLOW_NAME, schedule=get_cron_schedule(cron="0 0 15 * *")) as flow:
```

Comment:
FWIW we chose the 15th because it sounded nice and would run on a day when we are in the office for the first time. This would allow us to be present if there were any issues. G and I preferred this to a first automated run when everyone is on a long holiday weekend.

Reply:
Also, I have run this flow manually today and the deleted user account data has already been purged as of 9:30 AM pst